clients/web/src/views/widgets/UploadWidget.js

```diff
@@ -176,7 +176,15 @@ var UploadWidget = View.extend({
         this.$('.g-drop-zone')
             .removeClass('g-dropzone-show')
             .html(`<i class="icon-docs"/> ${this._browseText}`);
-        this.files = e.originalEvent.dataTransfer.files;
+
+        var dataTransfer = e.originalEvent.dataTransfer;
+
+        // Require all dropped items to be files
+        if (!_.every(dataTransfer.items, _.bind(this._isFile, this))) {
```

Comment:
Nowadays we favor using arrow functions instead of _.bind:

`_.bind(this._isFile, this)` -> `() => this._isFile()`